### PR TITLE
get application pods

### PR DESCRIPTION
**Pull request by kyleferguson**
Hooks into Johns work to query Kubernetes for application pods.

Started the ability for us to seed data. Need to finish some work on that so we can test all the way through. The ability to get pods for an application is code complete now though, I think :)

### CODE REVIEW

File: resources/pods.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package resources
```
Comment (jlaswell) [inline resources/pods.go:1]: Filename was pluralized. I don't see a problem with that, but it should be consistent between resources.

**Review by jlaswell — APPROVED** [review]
Not 100% on the kube functionality in the pods section, but this looks good for everything else.

File: resources/pods.go (hunk 1 above)
Comment (kyleferguson) [reply resources/pods.go:1]: Was an accident. I moved it to a new directory and ended up moving it back and didn't pay attention to the other resource

**Comment by kyleferguson** [comment]
I have some ideas for refactoring and testing. Lets get together tomorrow and review it. I'm going to merge so I can test everything out together and see where we are.

**Pull request merged**